.github/workflows/test_development_environment.yml

```diff
@@ -15,10 +15,10 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
-        node: [12, 13, 14]
+        node: [16, 18]
```

Comment:
this is to test that the code is compatible with future node version. as such. no need to test for 12 13 14 since we're upgrading to 16. no need to test for 17 also because we always use even versions of node